From 5c33a2cff7e1fcd3642e5f32320ecc2b88da188c Mon Sep 17 00:00:00 2001
From: fli <fupan.li@windriver.com>
Date: Wed, 24 Aug 2016 00:33:57 -0700
Subject: [PATCH] smartpm: make smart send system id to rpm repo server

Signed-off-by: fli <fupan.li@windriver.com>
---
 smart/fetcher.py | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/smart/fetcher.py b/smart/fetcher.py
index 6537546..deea799 100644
--- a/smart/fetcher.py
+++ b/smart/fetcher.py
@@ -727,6 +727,12 @@ class URL(object):
         self.passwd = self.passwd and urllib.unquote(self.passwd) or ""
         self.path = urllib.unquote(self.path)
 
+        if os.path.isfile("/etc/system-id"):
+            fileid = open("/etc/system-id")
+            systemid = fileid.readline().strip('\n')
+
+        data = urllib.urlencode({"id":systemid})
+        self.query = data
         opener = Opener()
         remote = opener.open(str(self))
         cached = 1
-- 
2.7.4

